<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectQtDevToolsDialog</class>
 <widget class="QDialog" name="DetectQtDevToolsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Qt path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="QtRootPathEdit">
       <property name="placeholderText">
        <string>detected if empty</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BrowseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="RunButton">
     <property name="text">
      <string>Run detection...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="DetectionProcessTextEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
